app/modules/entities/components/cleanup/lib/get_parts_suggestions.ts

```diff
@@ -0,0 +1,59 @@
+import { uniq, property, pluck, flatten } from 'underscore'
+import { API } from '#app/api/api'
+import preq from '#app/lib/preq'
+import { getEntities, type SerializedEntity } from '#app/modules/entities/lib/entities'
+import { searchWorks } from '#entities/lib/search/search_by_types'
+import type { GetAuthorWorksResponse } from '#server/controllers/entities/get_entity_relatives'
```

Comment:
I didn't know this was authorised ! Is it temporary ? Isn't the door opening to import whatever code (beyond types) from the server to client ?

Reply:
Used strictly for types, see https://github.com/search?q=repo%3Ainventaire%2Finventaire-client%20%23server%2Fcontrollers&type=code